### PR TITLE
Change deprecated psr 2 to psr 12 from doc

### PR DESCRIPTION
**Pull request by woprrr**
PSR 2 => PSR 12

### CODE REVIEW

File: contributing/code/standards.rst [hunk 1]
```diff
@@ -5,7 +5,7 @@ Symfony code is contributed by thousands of developers around the world. To make
 every piece of code look and feel familiar, Symfony defines some coding standards
 that all contributions must follow.
 
-These Symfony coding standards are based on the `PSR-1`_, `PSR-2`_ and `PSR-4`_
+These Symfony coding standards are based on the `PSR-1`_, `PSR-12`_ and `PSR-4`_
```
Comment (BackEndTea) [inline contributing/code/standards.rst:8]: PSR-12 builds on top op PSR-2, just like PSR-2 builds on top of PSR-1. So we should probably keep PSR-2.

Comment (BackEndTea) [reply contributing/code/standards.rst:8]: The link to PSR-12 is missing right now.

Comment (javiereguiluz) [reply contributing/code/standards.rst:8]: Fixed while merging. Thanks!

**Comment by javiereguiluz** [comment]
Alexandre, thanks and congrats on your first Symfony Docs contribution 🎉

**Comment by woprrr** [comment]
Thanks @javiereguiluz glad to help !

**Pull request merged**